modules/administration-guide/pages/installing-che-on-minikube.adoc

```diff
@@ -18,6 +18,4 @@ include::partial$proc_using-minikube-to-set-up-kubernetes.adoc[leveloffset=+1]
 
 include::partial$proc_installing-che-on-minikube-using-chectl.adoc[leveloffset=+1]
```

Comment:
You can move the content of the file here, and remove the partial.

Reply:
And also merge the one step procedure in proc_using-minikube-to-set-up-kubernetes.adoc here.

Reply:
is it optional?

Reply:
in general, I will let the docs team to do the content reorg, this is not smth that I plan to do as part of this PR